src/components/buttons/RadioButton/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import React from 'react';
+import styles from "./radioButton.module.css";
+import { RenderOptions } from './RenderOptions';
+
+interface IOption {
+    id: string;
+    label: string;
+    disabled: boolean;
+}
+
+interface RadioButtonGroupProps {
+    id: string;
+    label: string;
+    options: IOption[];
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
I'm missing selected or a default selected, ref: https://medium.com/@christinaroise/how-to-create-a-reusable-custom-radiobutton-in-react-with-typescript-3ae7fc704e09

With radio groups there has to be one option pre-selected. 

We can set this in two ways:
- add an option prop: selectedOption?: IOption;
- set selected option to selectedOption || options[0];